cmd/podman/create_cli.go

```diff
@@ -98,6 +98,24 @@ func parseVolumes(volumes []string) error {
 	return nil
 }
 
+func parseVolumesFrom(volumesFrom []string) error {
+	if len(volumesFrom) == 0 {
+		return nil
+	}
+	for _, vol := range volumesFrom {
+		arr := strings.SplitN(vol, ":", 2)
+		if len(arr) == 2 {
+			if strings.Contains(arr[1], "Z") {
+				return errors.Errorf("invalid options %q, can only specify 'ro', 'rw', and 'z", arr[1])
```

Comment:
Do you need similar guards for `rprivate`, etc.?  It might be easier for us, and more powerful for users, to just allow everything that `validateVolumeOpts` supports and punt to its docs for `--volumes-from`.

Reply:
Added the guards for the other options. I know we definitely don't want to support `Z` as changing the label to private in the second container will cause the first container not to be able to access it. `private`, `shared`, and `slave` would probably cause similar issues and docker run doesn't support it either.